File: hellocharts/src/main/java/net/raquezha/lecho/hellocharts/animation/ChartDataAnimator.java
package net.raquezha.lecho.hellocharts.animation;

public interface ChartDataAnimator {

    long DEFAULT_ANIMATION_DURATION = 500;

    void startAnimation(long duration);

    void cancelAnimation();

    @SuppressWarnings("unused")
    boolean isAnimationStarted();

    void setChartAnimationListener(ChartAnimationListener animationListener);

}

File: hellocharts/src/main/java/lecho/lib/hellocharts/animation/ChartDataAnimator.kt
package lecho.lib.hellocharts.animation

interface ChartDataAnimator {
    fun startAnimation(duration: Long)
    fun cancelAnimation()
    val isAnimationStarted: Boolean
    fun setChartAnimationListener(animationListener: ChartAnimationListener?)

    companion object {
        const val DEFAULT_ANIMATION_DURATION: Long = 500
    }
}